src/main/resources/templates/mail/listOfUnapprovedUsers.html

```diff
@@ -23,17 +24,52 @@
     <th>Job Title</th>
     <th>Company</th>
     <th>Country</th>
+    <th>View Request in Slack</th>
   </tr>
   </thead>
   <tbody>
-  <tr th:each="user: ${users}">
+  <tr th:each="user: ${undiscussedUsers}">
```

Comment:
Haven't used before, but we can consider create a fragment for the table so it could be reused. https://www.baeldung.com/spring-thymeleaf-fragments